app/controllers/api/v1/stats/channels_controller.rb

```diff
@@ -0,0 +1,48 @@
+class Api::V1::Stats::ChannelsController < Api::BaseController
+
+def channels
+  channels = Channel.all.map { |channel| channel.id }
+  data = JSON.pretty_generate(channels)
+
+  render(json: data)
+
+end
+
+def channels_uuid
+
+  channel = Channel.find_by_id(params[:uuid])
+
+  case channel.details_type
+  when "SiteChannelDetails"
+    channel_details = SiteChannelDetails.find_by_id(channel.details_id)
+    platform = "Site"
```

Comment:
promo uses "website"
https://github.com/brave-intl/publishers/blob/8b6093cb8fc8487534e1e898d0a224ef00a0166c/app/services/promo_registrar.rb#L120
 and also "channel_type" instead of platform

Reply:
Will change to lowercase and `channel_type`
so:

`channel_type` : `website`
`channel_type` : `youtube`
`channel_type` : `twitter`
`channel_type` : `twitch`

please confirm

Reply:
Great, fixed